src/Compatibility/Microsoft.DotNet.ApiCompatibility/Abstractions/MetadataInformation.cs

```diff
@@ -29,6 +30,13 @@ public MetadataInformation(string name, string targetFramework, string assemblyI
             string.Equals(TargetFramework, other.TargetFramework, StringComparison.OrdinalIgnoreCase) &&
             string.Equals(AssemblyId, other.AssemblyId, StringComparison.Ordinal);
 
-        public override int GetHashCode() => HashCode.Combine(AssemblyName, TargetFramework, AssemblyId);
+        public override int GetHashCode()
```

Comment:
Why did we need to do this change?

Reply:
This is in order to remove the Microsoft.Bcl.HashCode dependency and this is in custom with what sdk is doing at other places.